mongomock/helpers.py

```diff
@@ -7,8 +7,8 @@
 
 
 try:
-    from bson import (ObjectId, RE_TYPE)
-    from bson.tz_util import utc
+    from bson import (ObjectId, RE_TYPE, tz_util)
```

Comment:
This still suffers from the same problem - if only RE_TYPE is missing, you will use your own ObjectId....

Reply:
I'm not sure to understand you... originally the try/except contains only `from bson import (ObjectId, RE_TYPE)`.
So the fact that the custom version of ObjectId will be used if we cannot import RE_TYPE doesn't come from my PR right ? (not pointing any finger here, it's just to understand :smile: )

I would say it's not a big concern given the bson module is pretty stable so the try/except guard is only to fallback if the module is not present, not to allow mixed configuration when only a part of the module is present.

Anyway, if you prefer I can modify the PR to do the try/except per element imported:
```python
try:
    from bson import ObjectId
except ImportError:
    class ObjectId:
        # whatever

try:
    from bson import RE_TYPE
except ImportError:
    RE_TYPE = ...

try:
    from bson.tz_util import utc
except ImportError:
    ...
```

What do you say ? :stuck_out_tongue:

Reply:
You're right - it wasn't introduced in this PR - so not holding any blame here :smile: 

I would very much like to see the change you included above. I think it will make the code better. Thanks!

Reply:
I've updated the PR :smiley: